swagger_server/models/business.py

```diff
@@ -1,12 +1,11 @@
 from sqlalchemy import Column
 from sqlalchemy.orm import relationship
 from sqlalchemy.types import Integer, String
-from ..configuration import ons_env
 from .associations import instrument_business_table
-from .base import Base
+from ons_ras_common import ons_env
 
 
-class BusinessModel(Base):
+class BusinessModel(ons_env.db.base):
```

Comment:
I still don't see why it's necessary to attach the model base class to ons_env. Not that it has much to do with this PR.